pkg/driver/controller_test.go

```diff
@@ -410,6 +427,8 @@ func TestCreateVolume(t *testing.T) {
 						volumeParamsSecurityGroupIds:              securityGroupIds,
 						volumeParamsSubnetIds:                     subnetIds,
 						volumeParamsTags:                          tags,
+						volumeParamsOptionsOnDeletion:             optionsOnDeletion,
+						volumeParamsSkipFinalBackupOnDeletion:     skipFinalBackupOnDeletion,
 						"key":                                     "value",
```

Comment:
I'm actually not sure how this method passes. We should look into if it still does and remove it. I may have forgotten to remove it when I removed the check.

Reply:
It was passing because it was failing for a different reason; the test originally didn't provide skipFinalBackup lol